toffy/streak_detection.py

```diff
@@ -346,7 +352,7 @@ def _correct_mean_alg(
             input_image[max_row + 1, min_col + 1: max_col + 1],
         ],
         axis=0,
-        dtype=np.uint8,
+        dtype=input_image.dtype,
```

Comment:
In general, it's bad practice to hardcode the dtype. This line caused the float data being read in to be converted to ints, throwing away most of the values